kube-metrics-adapter/templates/deployment.yaml

```diff
@@ -49,14 +53,24 @@ spec:
         volumeMounts:
         - mountPath: /tmp
           name: tmp
+        - name: ssl-certs
+          mountPath: {{ .Values.sslCertPath }}
+          readOnly: true
     {{- if .Values.tolerations }}
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
     {{- end }}
     {{- if .Values.affinity }}
       affinity:
 {{ toYaml .Values.affinity | indent 8 }}
+    {{- end }}
+    {{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
     {{- end }}
       volumes:
       - name: tmp
         emptyDir: {}
+      - name: ssl-certs
```

Comment:
Could you please make this option as well `if .Values.aws.enable`

Reply:
Done, Thank you for the review.